airflow/dags/create_external_tables/sentry_error_logs.yml

```diff
@@ -0,0 +1,42 @@
+operator: operators.ExternalTable
+bucket: "{{ env_var('CALITP_BUCKET__SENTRY_EVENTS') }}"
+suffix_bucket: true
+source_objects:
+  - "*.jsonl.gz"
+destination_project_dataset_table: "external_sentry_error_logs.events"
+source_format: NEWLINE_DELIMITED_JSON
+use_bq_client: true
+hive_options:
+  mode: CUSTOM
+  source_uri_prefix: "events/{dt:DATE}/{ts:TIMESTAMP}/"
```

Comment:
Do you have thoughts on partitioning this by event date rather than extract date? Otherwise if we run a backfill, we'll end up with duplicate data rather than overwriting our already-extracted data.

I think this actually dovetails with our payments v2 discussion as well, in that we have to deal with the potential for multiple extracts of the same historical data depending on backfills, retries, etc.